core/src/apps/tron/providers.py

```
@@ -0,0 +1,25 @@
+from typing import Iterator
```

Ruff (0.14.5)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)